### PR TITLE
Prevent refetching of permissions

### PR DESCRIPTION
**Pull request by ZachCMP**
https://www.pivotaltracker.com/story/show/184630508

### CODE REVIEW

File: src/api/operations/clients.queries.graphql [hunk 1]
```diff
@@ -34,6 +34,7 @@ query GetClientName($id: ID!) {
 
 query GetClientPermissions($id: ID!) {
   client(id: $id) {
+    id
     ...ClientPermissions
```
Comment (gigxz) [inline src/api/operations/clients.queries.graphql]: `ClientPermissions` already has `id` on it, so I don't think this is doing anything

File: src/modules/permissions/PermissionsFilters.tsx [hunk 2]
```diff
@@ -79,6 +79,7 @@ export const ClientPermissionsFilter: React.FC<
 > = ({ id, ...props }) => {
   const { data, loading } = useGetClientPermissionsQuery({
     variables: { id },
+    skip: !id,
```
Comment (gigxz) [inline src/modules/permissions/PermissionsFilters.tsx:82]: When is this happening? It looks like a case of us expecting `clientId` to be present in the route, but not finding it?

**Comment by gigxz** [comment]
Hmm, this isn't fixing the repeated requests for me. I'm still seeing repeated requests as I click around the dashboard for `GetClientPermissions` for the same client ID. I thought Apollo would save us from that! I'm seeing this warning in the console.


```
Cache data may be lost when replacing the access field of a Client object.

To address this problem (which is not a bug in Apollo Client), either ensure all objects of type ClientAccess have an ID or a custom merge function, or define a custom merge function for the Client.access field, so InMemoryCache can safely merge these objects:

  existing: {"__typename":"ClientAccess","canViewDob":true,"canViewFullSsn":true,"canViewPartialSsn":false,"canEditEnrollments":true,"canDeleteEnrollments":true,"canViewEnrollmentDetails":true,"canManageAnyClientFiles":true,"canManageOwnClientFiles":true,"canViewAnyConfidentialClientFiles":true,"canViewAnyNonconfidentialClientFiles":true}
  incoming: {"__typename":"ClientAccess","canViewFullSsn":true,"canViewPartialSsn":false}

For more information about these options, please refer to the documentation:

  * Ensuring entity objects have IDs: https://go.apollo.dev/c/generating-unique-identifiers
  * Defining custom merge functions: https://go.apollo.dev/c/merging-non-normalized-objects
```

**Comment by ZachCMP** [comment]
Now relies on backend PR https://github.com/greenriver/hmis-warehouse/pull/2932 for the access type IDs

**Comment by ZachCMP** [comment]
@gigxz I'm not seeing the overfetching behavior now. This branch also needed a lot merged in from `release-70`, which I've done. It's possible something in there fixed the issue.

**Comment by gigxz** [comment]
Cool. I tried merging in the release-70 changes when I tested with your branch yesterday, and it didn't fix the issue for me. So, sounds like your backend change fixed it. It works for me. thanks!

**Pull request merged**